test/backward-stacktrace.h

```diff
@@ -0,0 +1,42 @@
+#include <csignal>
+
+#include "backward.hpp"
+#include "glog/logging.h"
+#include "gtest/gtest.h"
+
+class BackwardStackTrace : public ::testing::Environment {
```

Comment:
Please add a comment documenting this class. (Why do we declare it in a header, rather than in the one .cc file that uses it?)